en/docs/guides/organization-login/try-organization-login.md

```diff
@@ -153,43 +209,31 @@ For the Car Traders organization, [create the user](../../org-user-management) S
-        `http://localhost:3000/api/auth/callback/wso2is`
-    -   Add the below URL as the allowed origin.
-        `http://localhost:3000`
+Save the file and restart the Identity Server.
 
-### Step 3: Start the sample application
+## Try it out
 
-1.  Open a terminal, navigate to  the `<SAMPLE_APP_HOME>/guardio-insurance-sample-application-main` folder and run the following commands respectively to start the application:
+To try out this application:
 
+1. Open a terminal, navigate to `<SAMPLE_APP_HOME>/guardio-insurance-sample-application-main` folder and run the following commands to start the application:
+    
     !!! info
-        To complete this step, it is required to have node installed in your machine.
-
+        To complete this step, it is required to have a node installed in your machine.
+        
     ``` node
     npm install
     npm run dev
     ```
 
-2.  To check the sample application, go to `http://localhost:3000` on your browser.
-
-3.  Click **Sign In** to proceed in the application.
-
-    <img src="../../../assets/img/guides/organization-login/try-it-out/sample_app_sign_in.png" alt="Sample App Sign-in" width="700" style="border:1px solid grey">
-
-4.  Select an organization from the list and click **Next**. In this example, we will select Car Traders.
-
-    <img src="../../../assets/img/guides/organization-login/try-it-out/sample_app_org.png" alt="Sample App Orgs" width="700" style="border:1px solid grey">
-
-5.  Enter the user credentials to log in to the Car Traders Organization.
+2. Enter `http://localhost:3000` on your browser to access the application.
 
-    <img src="../../../assets/img/guides/organization-login/try-it-out/sign_in_car_traders_org.png" alt="Sign-in Car Traders Organization" width="700" style="border:1px solid grey">
+3. Click **Sign In** to proceed with the application.
 
-6.  Now, you have successfully logged in to the sample application.
+4. Select your organization from the list and click **Next**. In this example, we will select `Car Traders` as the organization.
 
-    <img src="../../../assets/img/guides/organization-login/try-it-out/logged_in_to_sample_app.png" alt="Logged in to Sample App" width="700" style="border:1px solid grey">
+    <img src="../../../assets/img/guides/organization-login/try-it-out/sample_app_org.png" alt="Sample App Orgs" width="700" style="border:0px solid grey">
 
-7.  Now, select Best Auto Mart Organization and try to log in. You’ll be redirected to the Asgardeo IdP based login page.
+5. Enter the user credentials and click **Continue** to log in to the application.
+    You will be successfully logged in to the sample application.
 
-    <img src="../../../assets/img/guides/organization-login/try-it-out/asgardeo_based_login.png" alt="Asgardeo based Login" width="400" style="border:1px solid grey">
+6. Logout of the application, select `Best Auto Mart` organization and try to log in. You will be redirected to the Asgardeo login page.
```

Comment:
If we are going with the terms "sign in" and "sign out", we should consider using the same throughout the doc.